client/state/signup/steps/business-details/reducer.js

```diff
@@ -0,0 +1,44 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+
+import {
+	SIGNUP_COMPLETE_RESET,
+	SIGNUP_STEPS_BUSINESS_ADDRESS_SET,
+	SIGNUP_STEPS_BUSINESS_NAME_SET,
+} from 'state/action-types';
+
+import { createReducer, combineReducers } from 'state/utils';
+
+const address = createReducer(
```

Comment:
I'd discourage `createReducer` for simple reducers like these where where a `switch` handles the case nicely and idiomatically:

```js
function address( state = '', action ) {
  switch ( action.type ) {
    case SIGNUP_STEPS_BUSINESS_ADDRESS_SET:
      return action.businessAddress;
    case SIGNUP_COMPLETE_RESET:
      return '';
  }
  return state;
}
address.schema = { type: 'string' };
```